tensorflow/lite/micro/micro_allocation_info.cc

```diff
@@ -245,6 +241,9 @@ TfLiteStatus AllocationInfoBuilder::MarkAllocationLifetimes(
     const int tensor_index = subgraph->inputs()->Get(i);
     AllocationInfo* current = &subgraph_allocation_info[tensor_index];
     UpdateFirstCreated(current, allocation_scope_count_);
+    // This will ensure that tensors that are outputs from an
+    // OP but not inputs to any other OP also have a reasonable lifetime.
```

Comment:
is this comment accurate? I would expect to say something along the lines of tensors that are inputs to the subgraphs but not used in any ops (see b/<> for an example of when this can happen).

Reply:
Done.